src/pages/stats/index.tsx

```diff
@@ -52,9 +52,9 @@ export default function Stats() {
   }
 
   return (
-    <>
+    <div data-testid="stats-grid" className="h-full">
```

Comment:
Is adding `.h-full` related to tests?

Reply:
nvm I see this is because it used to be a fragment. I trust this will render exactly the same?